_plugins/changelog.rb

```diff
@@ -0,0 +1,59 @@
+require 'open3'
+require 'byebug'
```

Comment:
I think this might need to be added to the gemfile or something.

Reply:
`open3` is a standard lib installed by default and `byebug` is an interactive debugger useful just for development -- no need to change the `Gemfile`. :wink:

Reply:
ah ok, i wasn’t sure what the byebug thing was and i didn’t have it installed. no worries!